action.yml

```diff
@@ -40,4 +42,5 @@ runs:
     - --test_suite_id=${{ inputs.test_suite_id }}
     - --environment_id=${{ inputs.environment_id }} 
     - --wait=${{ inputs.wait }}
+    - --commit_sha={{ inputs.commit_sha }}
```

Comment:
Can we try reading it from `GITHUB_SHA`?

https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables

A few more env vars seem relevant as well, like `GITHUB_REF` and `GITHUB_REPOSITORY`

Wondering if we can just pass in all the env vars prefixed by `GITHUB` to the run context

Reply:
I will try this out properly when testing with Staging Active Monitor